runtime-modules/content/src/tests/creator_tokens/claim_patronage_credit.rs

```diff
@@ -93,3 +96,17 @@ fn claim_creator_token_patronage_credit_fails_during_trasfer() {
             .call_and_assert(Err(Error::<Test>::InvalidChannelTransferStatus.into()));
     })
 }
+
+#[test]
+fn claim_member_channel_creator_token_patronage_credit_by_owner_fails_on_frozen_pallet() {
+    with_default_mock_builder(|| {
+        ContentTest::with_member_channel().setup();
+        IssueCreatorTokenFixture::default().call_and_assert(Ok(()));
+
+        let freeze_result = Token::set_frozen_status(RawOrigin::Root.into(), true);
+        assert_ok!(freeze_result);
+
+        ClaimCreatorTokenPatronageCreditFixture::default()
+            .call_and_assert(Err(project_token::Error::<Test>::PalletFrozen.into()));
+    })
```

Comment:
For all the new tests being added, since you already are doing the setup and testing the call fails if pallet is frozen, you should follow it up with a call to unfreeze the pallet and then ensure that the original fixture/call succeeds.